pom.xml

```diff
@@ -553,6 +576,11 @@
                                     -javaagent:${settings.localRepository}/org/jacoco/org.jacoco.agent/${jacoco.version}/org.jacoco.agent-${jacoco.version}-runtime.jar=destfile=${session.executionRootDirectory}/target/jacoco-oracle.exec
                                 </argLine>
                             </argLines>
+                            <environmentVariables>
```

Comment:
Shouldn't it be added to all the profiles? In this case, it is only in the Oracle's profile.

Reply:
yes, you are right, that explains a lot :)

Reply:
But anyway I can send this change in a different PR because I'm only testing with the normal J8 config in the pipeline this time to send the snapshot. If you don't mind, please waive the changes request on your review so I can proceed if the tests pass